--- components/chat.tsx
+++ components/chat.tsx
@@ -156,10 +156,7 @@
                   <EmptyScreen
                     submitMessage={message => {
                       setInput(message)
-                      setTimeout(() => {
-                        setIsSubmitting(true)
-                      }, 0)
+                      setIsSubmitting(true)
                     }}
                   />
                 ) : (
@@ -198,10 +195,7 @@
                   <EmptyScreen
                     submitMessage={message => {
                       setInput(message)
-                      setTimeout(() => {
-                        setIsSubmitting(true)
-                      }, 0)
+                      setIsSubmitting(true)
                     }}
                   />
                 ) : (
